corelib-schemaorg/src/main/java/eu/europeana/corelib/edm/model/schemaorg/Timespan.java

```diff
@@ -0,0 +1,68 @@
+package eu.europeana.corelib.edm.model.schemaorg;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldType;
+
+import java.util.List;
+
+@JsonldType(SchemaOrgConstants.TYPE_TIMESPAN)
+@JsonInclude(value = JsonInclude.Include.NON_EMPTY)
+@JsonPropertyOrder({"@type", "@id", 
+    SchemaOrgConstants.PROPERTY_NAME,SchemaOrgConstants.PROPERTY_ALTERNATE_NAME, SchemaOrgConstants.PROPERTY_DESCRIPTION,
+    SchemaOrgConstants.PROPERTY_SAME_AS})
+public class Timespan extends ContextualEntity {
+
+    @JsonIgnore
+    @Override
+    public String getTypeName() { return SchemaOrgConstants.TYPE_TIMESPAN; }

```

Comment:
according to the mapping, the SchemaOrg type for Timespans is Thing not timespan. please update and remove the JsonIgnore annotation

Reply:
updated!